rocky/assets/js/boefjeSetupRunOn.js

```diff
@@ -0,0 +1,36 @@
+function toggleElements(value) {
+  const intervalNumber = document.querySelector(
+    "label[for='id_interval_number']",
+  ).parentElement;
+  const intervalFrequency = document.querySelector(
+    "label[for='id_interval_frequency']",
+  ).parentElement;
+  const runOn = document.querySelector("label[for='id_run_on']").parentElement;
+
+  if (value === "interval") {
+    intervalNumber.classList.remove("hidden-input");
+    intervalFrequency.classList.remove("hidden-input");
+    runOn.classList.add("hidden-input");
+  } else if (value === "run_on") {
+    intervalNumber.classList.add("hidden-input");
+    intervalFrequency.classList.add("hidden-input");
+    runOn.classList.remove("hidden-input");
+  } else {
+    intervalNumber.classList.add("hidden-input");
+    intervalFrequency.classList.add("hidden-input");
+    runOn.classList.add("hidden-input");
+  }
+}
+
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
Make the internal anonymous function not anonymous. That way If we get to a situation where we want to add another set of form elements, we can just re-call the init function that sets up the event listeners.